modules/nf-core/rsem/preparereference/main.nf

```diff
@@ -69,4 +69,15 @@ process RSEM_PREPAREREFERENCE {
         END_VERSIONS
         """
     }
+
+    stub:
+    """
+    touch genome.transcripts.fa
```

Comment:
It should be a silly reason, but why is not complaining about not finding the `index` when running the stub?

Reply:
Ah true, it was a silly reason, should have looked into the `input` 😄
Thanks!